fastplotlib/graphics/_features/_deleted.py

```diff
@@ -0,0 +1,41 @@
+from ._base import GraphicFeature, FeatureEvent
+
+
+class Deleted(GraphicFeature):
+    """
+    Used when a graphic is deleted, triggers events that can be useful to indicate this graphic has been deleted
+
+    **event pick info:**
+
+     ==================== ======================== =========================================================================
+      key                  type                     description
+     ==================== ======================== =========================================================================
+      "collection-index"   int                      the index of the graphic within the collection that triggered the event
+      "world_object"       pygfx.WorldObject        world object
+     ==================== ======================== =========================================================================
+    """
+
+    def __init__(self, parent, value: bool):
+        super(Deleted, self).__init__(parent, value)
+
+    def _set(self, value: bool):
+        value = self._parse_set_value(value)
+        self._feature_changed(key=None, new_data=value)
+
+    def _feature_changed(self, key, new_data):
+        # this is a non-indexable feature so key=None
+
+        pick_info = {
+            "index": None,
+            "collection-index": self._collection_index,
```

Comment:
What is the difference here between `index` vs `collection-index`? Also, `collection-index` could be `None` if the graphic is not a `CollectionGraphic`, no?

Reply:
It seems like `index` will always be `None`?

Reply:
index is buffer index, used for things like `data` and `colors` features. Not used here since it's not relevant